frontend/assets/stylesheets/layout/_layout.scss

```diff
@@ -56,6 +56,14 @@ body {
     }
 }
 
+.l-gutter {
+    padding: rem($gs-baseline) rem($gs-gutter) rem($gs-baseline*2);
```

Comment:
Still not quite sure about this, given the name it feels like this should only be adding gutters (e.g., left/right padding). Could this be renamed perhaps?